src/service/wallet.ts

```diff
@@ -119,14 +119,19 @@ export const makeWalletService = () => {
         case Errors.enableKeplr:
           showToast('Enable the connection in Keplr to continue.', {
             hideProgressBar: false,
-            autoClose: 5000,
+            autoClose: 7000,
           });
           break;
         case Errors.networkConfig:
           showToast('Network not found.');
           break;
         case Errors.noSmartWallet:
-          showToast(React.createElement(SmartWalletNotFoundToast, null));
+          showToast(
+            React.createElement(SmartWalletNotFoundToast, {
```

Comment:
what would it take to make the message disappear once the smart wallet is made?

Reply:
Also I'd like to move towards the end goal of an extension subsuming wallet.agoric.app (metamask snap, frontier etc.) sooner than later, and this is tangential to that.